birdclef/workflows/evaluation.py

```diff
@@ -0,0 +1,121 @@
+from birdclef.models.embedding.tilenet import TileNet
+import pandas as pd
+from birdclef.workflows import motif
+from pathlib import Path
+import IPython.display as ipd
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from sklearn.decomposition import PCA
+from sklearn.cluster import KMeans
+from sklearn.preprocessing import LabelEncoder
+from jinja2 import Environment, FileSystemLoader
+import webbrowser
+import click
+import os
+from datetime import date
+
+
+@click.group()
+def evaluation():
+    pass
+
+
+@evaluation.command()
+@click.option('--intra', default="brnowl", help='Singular species to perform intraclass inspection')
+@click.option('--inter', default="brnowl,skylar,norcar", help='Array of 3 species to perform interclass comparison')
+@click.option(
+    "--checkpoint",
+    type=click.Path(exists=True, dir_okay=False),
+    default=Path(
+        "../../data/processed/model/2022-04-12-v4/embedding.ckpt"
```

Comment:
2022-04-12-v4 is old and won't work with current versions of the code, you'll need to use a more recent model.